include/engine/datafacade/contiguous_internalmem_datafacade.hpp

```diff
@@ -154,7 +155,7 @@ class ContiguousInternalMemoryAlgorithmDataFacade<algorithm::CoreCH>
     : public datafacade::AlgorithmDataFacade<algorithm::CoreCH>
 {
   private:
-    util::ShM<bool, true>::vector m_is_core_node;
+    util::ShM<unsigned, osrm::storage::MemorySetting::SharedMemory>::vector m_is_core_node;
```

Comment:
i think the problem is here because `unsigned` could have different sizes.

Could you clarify why `bool` was changed to `unsigned`?